src/components/datatable/DocumentsTable.vue

```diff
@@ -46,6 +48,25 @@
     return Object.assign(result, options);
   }
 
+  // comparator functions for custom sort of "complicated" columns
+  // these functions could be removed if the sort function of
+  // AG grid could be deactivated since documents are expected to
+  // be given in correct order by the backend
+  function condComp(cond1, cond2) {
+    const conds = { 'excellent': 0, 'good': 1, 'average': 2, 'poor': 3, 'awful': 4 };
```

Comment:
You can use array defined in `/src/js/constants/common.json` as a source for your map. It does not duplicate this list and is more maintenable.

Same for other comparator